### PR TITLE
Fix LinkedIn icon bug #10

### PR DESCRIPTION
**Pull request by nparashar150**
**Fixed LinkedIn icon bug**
Replaced it from Font Awesome icon --> custom .svg icon.

**Expected Behavior**
Screenshot from 2021-05-24 10-10-41

### CODE REVIEW

File: users/static/users/css/login.css [hunk 1]
```diff
@@ -150,31 +152,47 @@ body {
...
+@keyframes main-logo-animation {
+  0% {
+    filter: invert(27%) sepia(51%) saturate(4000%) hue-rotate(300deg) brightness(100%) contrast(100%);
+  }
+  50% {
+    filter: invert(27%) sepia(51%) saturate(4000%) hue-rotate(200deg) brightness(100%) contrast(100%);
+  }
+  100% {
+    filter: invert(27%) sepia(51%) saturate(4000%) hue-rotate(300deg) brightness(100%) contrast(100%);
+  }
```
Comment (ansh-saini) [inline users/static/users/css/login.css:179]: This breaks the rotate animation on firefox.
Plus, we don't really want the background color to change like that.
Please revert this change.

File: users/static/users/css/login.css [hunk 2]
```diff
@@ -150,31 +152,47 @@ body {
...
 }
 
 .login {
   border-radius: 3rem;
   width: 35rem;
   height: 34rem;
-  background-color: rgba(255, 255, 255, 0.1);
+  background: rgb(0,119,181);
+  background: linear-gradient(45deg, rgba(0,119,181,0.2) 0%, rgba(255,102,159,0.2) 100%);
```
Comment (ansh-saini) [inline users/static/users/css/login.css:187]: Don't need a gradient here. Please revert this change.

File: users/static/users/images/Facebook Logo.svg [hunk 3]
```diff
@@ -1 +1,40 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (ansh-saini) [inline users/static/users/images/Facebook Logo.svg:39]: So many empty `g` elements?

File: users/static/users/images/Linked Logo.svg [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<svg width="512" height="512" xmlns="http://www.w3.org/2000/svg">
```
Comment (ansh-saini) [inline users/static/users/images/Linked Logo.svg:1]: Rename this file to `Linkedin Logo.svg`

File: users/static/users/css/login.css [hunk 5]
```diff
@@ -115,11 +115,12 @@ body {
   left: 4rem;
   cursor: pointer;
   background-color:#d34836;
+  transition: all .3s ease-in-out;
 }
 
 .btn-google:hover {
   transform: translateY(-0.2rem);
-  box-shadow: 0 0.5rem 0.5rem rgba(0, 0, 0, 0.5);
+  filter: drop-shadow(0 .3rem .3rem #7e0f0083);
```
Comment (ansh-saini) [inline users/static/users/css/login.css:123]: This is nice! 🤩

**Review by ansh-saini — CHANGES_REQUESTED** [review]

File: users/static/users/css/login.css (hunk 5 above)
Comment (nparashar150) [reply users/static/users/css/login.css:123]: Thank You :)

File: users/static/users/images/Facebook Logo.svg (hunk 3 above)
Comment (nparashar150) [reply users/static/users/images/Facebook Logo.svg:39]: This was just the avg generated, I didn't made it myself.

File: users/static/users/css/login.css (hunk 2 above)
Comment (nparashar150) [reply users/static/users/css/login.css:187]: No worries.

**Pull request closed without merging**